.github/workflows/ci.yml

```diff
@@ -86,7 +86,15 @@ jobs:
           path: ~/.npm
           key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
       - run: npm ci
-      - run: npm run test:docker
+      - run: npm run test:docker      
+      - name: Configure Datadog Test Visibility
+        env:
+          DD_API_KEY: ${{ secrets.DD_API_KEY }}
+        uses: datadog/test-visibility-github-action@v1
+        with:
+          languages: js
+          service: isomer
+          api_key: ${{ secrets.DD_API_KEY }}
       - run: . .env.test && npx jest --runInBand
```

Comment:
need to add the `NODE_OPTIONS` env var to the test step so dd agent can inject itself

```
  env:
    NODE_OPTIONS: -r ${{ env.DD_TRACE_PACKAGE }}
```

Reply:
added